src/Symfony/Component/Messenger/Stamp/DelayStamp.php

```diff
@@ -11,11 +11,21 @@
 
 namespace Symfony\Component\Messenger\Stamp;
 
+use Symfony\Component\Messenger\Exception\InvalidArgumentException;
+
 /**
  * Apply this stamp to delay delivery of your message on a transport.
  */
 final class DelayStamp implements StampInterface
 {
+    public const PERIOD_SECONDS = 'seconds';
```

Comment:
Please remove

Reply:
Is this related to your comment about `delayFor()`?

Reply:
Yes, we dont need a way to redefine time.